### PR TITLE
Update Wiz vuln data stream for CDR

### PR DESCRIPTION
**Pull request by CohenIdo**
solves:
- https://github.com/elastic/security-team/issues/9829

## Summary
* Update vulnerability data stream's ingest pipeline to support CDR.
* Update vulnerability data stream's mappings according to the new fields.

### CODE REVIEW

File: packages/wiz/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.1"
```
Comment (efd6) [inline packages/wiz/changelog.yml:2]:
```suggestion
- version: "1.7.0"
```
since this is an enhancement.

File: packages/wiz/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+  changes:
+    - description: Update vulnerability data stream
```
Comment (efd6) [inline packages/wiz/changelog.yml:4]: Can we have a more informative description? What is the update and how would a user know whether making the upgrade is important for them? Suggest something like...
```suggestion
    - description: Add support for CDR to vulnerability data stream.
```
or
```suggestion
    - description: Improve support for CDR in vulnerability data stream.
```

File: packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -563,6 +563,49 @@ processors:
       tag: set_pipeline_error_to_event_kind
       value: pipeline_error
       if: ctx.error?.message != null
+  - set:
+      field: vulnerability.package.fixed_version
+      tag: cdr
```
Comment (efd6) [inline packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml:568]: Tags should be pipeline-uniqe. They are used to identify failing processors within a pipeline, so if they are all the same, they do not add information with utility.

Comment (kcreddy) [reply packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml:568]: Example: https://github.com/elastic/integrations/blob/main/packages/prisma_access/data_stream/event/elasticsearch/ingest_pipeline/default.yml#L238

File: packages/wiz/data_stream/vulnerability/fields/ecs.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  external: ecs
+- name: vulnerability.score.base
+  external: ecs
+- name: vulnerability.reference
+  external: ecs
+- name: package.name
+  external: ecs
+- name: package.version
+  external: ecs
+- name: observer.vendor
```
Comment (kcreddy) [inline packages/wiz/data_stream/vulnerability/fields/ecs.yml:11]: We need not manually define the ECS fields, unless you want to change their existing datatype.
https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html

In this case, you can delete this file `ecs.yml` as they are all taking their pre-defined ECS datatypes.
See: https://github.com/elastic/integrations/pull/10790#discussion_r1726897248

Comment (CohenIdo) [reply packages/wiz/data_stream/vulnerability/fields/ecs.yml]: I see, thanks.

**Comment by CohenIdo** [comment]
Thanks for the review, @kcreddy, @efd6, I have resolved all your comments.

**Review by kcreddy — APPROVED** [review]

**Review by efd6 — APPROVED** [review]

**Pull request merged**